packages/chain-adapters/src/bitcoin/BitcoinChainAdapter.ts

```diff
@@ -19,48 +17,31 @@ import {
 import { bitcoin } from '@shapeshiftoss/unchained-client'
 import coinSelect from 'coinselect'
 import split from 'coinselect/split'
-import WAValidator from 'multicoin-address-validator'
 
 import { ChainAdapter as IChainAdapter } from '../api'
 import { ErrorHandler } from '../error/ErrorHandler'
 import {
   accountTypeToOutputScriptType,
   accountTypeToScriptType,
-  bnOrZero,
-  convertXpubVersion,
   getStatus,
   getType,
   toPath,
   toRootDerivationPath
 } from '../utils'
+import { ChainAdapterArgs, UTXOBaseAdapter } from '../utxo/UTXOBaseAdapter'
 
-export interface ChainAdapterArgs {
-  providers: {
-    http: bitcoin.api.V1Api
-    ws: bitcoin.ws.Client
-  }
-  coinName: string
-  chainId?: CAIP2
-}
-
-export class ChainAdapter implements IChainAdapter<ChainTypes.Bitcoin> {
-  private readonly providers: {
-    http: bitcoin.api.V1Api
-    ws: bitcoin.ws.Client
-  }
-
+export class ChainAdapter
```

Comment:
Consider naming this BitcoinChainAdapter or similar to avoid name conflict with api.ChainAdapter interface?

Reply:
I assume this would introduce breaking changes upstream since this was the name already in place.  But yea, I agree it would be better for a variety of reasons.

Reply:
Base interface is already being imported as `IChainAdapter`. I have personally always found our prefixing unnecessary and overly verbose, ie. `bitcoin/BitcoinChainAdapter.ts`, `export class BitcoinChainAdapter`. Everything is already structured and scoped under the appropriate chain name and where we are importing multiple chains, the importing follows: 
```
import * as bitcoin from './bitcoin'
import * as cosmos from './cosmossdk/cosmos'
import * as ethereum from './ethereum'
```

This reads really nicely as `new bitcoin.ChainAdapter` imo.

Reply:
Okay, so are going with no change for the time being?

Reply:
Regardless of official decision, I don't think it belongs in this PR regardless

Reply:
yep all good with me. thank @kaladinlight